Add "A month after EclipseCon, Adoptium Community day summary, and more" post (#205)
adoptium/blog.adoptium.net: content/blog/a-month-after-eclipsecon-adoptium-community-day-summary-and-more/index.md

---
title: A month after EclipseCon - Adoptium Community day summary, and more.
date: "2022-11-24T12:00:00+00:00"
author: carmendelgado
description: Our Community Manager gives us a summary of Eclipse Adoptium activities around the EclipseCon event.
tags:
  - "Adoptium"
  - "EclipseCon"
---

> A month after the EclipseCon. I, Carmen Delgado, Adoptium Community manager, want to share my experience attending the [EclipseCon 2022](https://www.eclipsecon.org/2022) edition in Ludwigsburg, Germany, from 24 to  27 October.  It was my third week in the role, and this is how I documented the experience.

Reviewer: ```suggestion
A month after EclipseCon. I, Carmen Delgado, Adoptium Community manager, want to share my experience attending the [EclipseCon 2022](https://www.eclipsecon.org/2022) edition in Ludwigsburg, Germany, from 24 to  27 October.  It was my third week in the role, and this is how I documented the experience.
```

## Monday, the 24th: Community Day

> This day I had the opportunity to meet most of my Eclipse co-workers during our breakfast at the hotel. We had a morning session where Tim Ellison, George Adams, and Shelley Lambert went through the Adoptium working group status and all the projects, and the community gave some feedback. You can read the presentation [here](Adoptium-Community-Day-EclipseCon2022.pdf).

Reviewer 2: Would it be helpful to add embedded links to more information about who Tim, George, and Shelley are? Or to describe them briefly?

Author: Added the same title they shared during the community day, thank for the suggestion.

> This day I had the opportunity to meet most of my Eclipse co-workers during our breakfast at the hotel. We had a morning session where Tim Ellison, George Adams, and Shelley Lambert went through the Adoptium working group status and all the projects, and the community gave some feedback. You can read the presentation [here](Adoptium-Community-Day-EclipseCon2022.pdf).

Reviewer: ```suggestion
On Monday, I had the opportunity to meet most of my Eclipse co-workers during our breakfast at the hotel. We had a morning session where Tim Ellison, George Adams, and Shelley Lambert went through the Adoptium working group status and all the projects, and the community gave some feedback. You can view the presentation [here](Adoptium-Community-Day-EclipseCon2022.pdf).
```

 - **[Eclipse Temurin](https://adoptium.net/temurin/releases/)**. the Flagship of the Adoptium projects, the brand was created to separate our main product from the Adoptium brand and is a RUNTIME acronym.  is the most popular and active. It is a well-tested Java SE binary available across multiple platforms. Consumable via website, API, packages, container images, installers, and redistributors. Signed for safe OS use and with a $ free-to-use open source license. According to our download metrics, we have over [300k downloads daily trends](https://dash.adoptium.net), but we cannot track who does the download due to GDPR. Our premise is that if users need and make the case we will keep building it, but every time we release, we need more time and infra support to maintain all the versions. Some ideas we got from the community:
    - Be more clear about promoting the advantages of using Temurin and certifications, compliance, and licenses.

>     - Define methods how to get the community and promote Temurin.

Author: This was a question from someone in the audience who asked how they can be informed about all the updates and releases and how the can access the final information, and we had a small discussion about possible solutions.

>     - Define methods how to get the community and promote Temurin.

Reviewer 3: Ok, I made a suggestion for that

>  - **[AQAvit](https://adoptium.net/aqavit/)**. Adoptium verification program offers tools and infrastructure to run a large internal pool of tests and analyze results, and it was created in collaboration with multiple vendors. Offering Tests for Quality, functional correctness, performance, scalability, security, and robustness. It could be considered the largest distribution of OpenJDK Binaries The next step for this project is to get enterprise users involved, and they can use our systems to test their builds by picking the part of the code and areas they want to test. The value added offered by this project is that it allows us to go thru all the builds and, generate a report TRSS, process test the binaries for consistency.  As this project implies a lot of infrastructure, we are looking for two kinds of collaboration support one is for automation to optimize triage time, and the other could be thru sponsorship.

Author: Shelley mentioned this during her AQAvit presentation, maybe I'm wrong, and she was talking about Temurin. @smlambert ?

>  - **[AQAvit](https://adoptium.net/aqavit/)**. Adoptium verification program offers tools and infrastructure to run a large internal pool of tests and analyze results, and it was created in collaboration with multiple vendors. Offering Tests for Quality, functional correctness, performance, scalability, security, and robustness. It could be considered the largest distribution of OpenJDK Binaries The next step for this project is to get enterprise users involved, and they can use our systems to test their builds by picking the part of the code and areas they want to test. The value added offered by this project is that it allows us to go thru all the builds and, generate a report TRSS, process test the binaries for consistency.  As this project implies a lot of infrastructure, we are looking for two kinds of collaboration support one is for automation to optimize triage time, and the other could be thru sponsorship.

Reviewer 4: Suggestion to improve this section:

Adoptium's verification program offers tools and infrastructure to run a large pool of tests and analyze results, and it was created in collaboration with multiple vendors. The AQAvit project tests for quality where quality criteria includes functional correctness, performance, scalability, security, and robustness. At Adoptium, AQAvit is used to test the quality of Temurin, which is considered one of the largest distributions of OpenJDK Binaries. The next step for AQAvit project is to get enterprise users involved.   ~~They can use our systems to test their builds by picking the part of the code and areas they want to test.~~ They can use the AQAvit framework and tools to add even more testing as part of their software supply chain, or contribute tests to AQAvit directly.  One value-add offered by AQAvit is that it allows us to test the binaries for consistency in their quality, go through the massive number of builds and generate summarized reports.   As this project implies a lot of effort and infrastructure, we are looking for several kinds of collaboration: working with researchers to further automate and optimize execution and triage time and through sponsorship respectively.

>  - **[AQAvit](https://adoptium.net/aqavit/)**. Adoptium verification program offers tools and infrastructure to run a large internal pool of tests and analyze results, and it was created in collaboration with multiple vendors. Offering Tests for Quality, functional correctness, performance, scalability, security, and robustness. It could be considered the largest distribution of OpenJDK Binaries The next step for this project is to get enterprise users involved, and they can use our systems to test their builds by picking the part of the code and areas they want to test. The value added offered by this project is that it allows us to go thru all the builds and, generate a report TRSS, process test the binaries for consistency.  As this project implies a lot of infrastructure, we are looking for two kinds of collaboration support one is for automation to optimize triage time, and the other could be thru sponsorship.

Author: Ok, here is the latest suggestion:
```suggestion
 - **[AQAvit](https://adoptium.net/aqavit/)**. Adoptium's verification program offers tools and infrastructure to run a large pool of tests and analyze results, and it was created in collaboration with multiple vendors. The AQAvit project tests for quality, where quality criteria include functional correctness, performance, scalability, security, and robustness. At Adoptium, AQAvit is used to test the quality of Temurin, which is considered one of the largest distributions of OpenJDK Binaries. The next step for the AQAvit project is to get enterprise users involved. They can use the AQAvit framework and tools to add even more testing as part of their software supply chain or contribute tests to AQAvit directly. One value-add offered by AQAvit is that it allows us to test the binaries for consistency in their quality, go through the massive number of builds and generate summarized reports. As this project implies a lot of effort and infrastructure, we are looking for several kinds of collaboration: working with researchers to further automate and optimize execution and triage time and through sponsorship respectively.
```

 - **[Temurin compliance](https://projects.eclipse.org/projects/adoptium.temurin-compliance)**. is responsible for obtaining, managing, and executing the Oracle Java SE Compatibility Kit (JCK) on Eclipse Temurin binaries. It is run as a non-open project and counts upon private infrastructure. The compliance project requires additional agreements in place for the community to participate. 

>  - **[Eclipse mission control](https://adoptium.net/jmc/)** Low-overhead monitoring and management of running Java applications. The idea is to get these services published as part of the Adoptium marketplace umbrella.

Reviewer: ```suggestion
 - **[Eclipse mission control](https://adoptium.net/jmc/)**. Low-overhead monitoring and management of running Java applications. The idea is to get these services published as part of the Adoptium marketplace umbrella.
```

 - **[Eclipse Adoptium Incubator](https://projects.eclipse.org/projects/adoptium.incubator)**. With this line, Adoptium wants to foster collaboration with a low threshold for setting up the project. The main goal is to prompt innovation, allowing the community to explore. Currently, we have a project about a [Fast Start-up](https://blog.adoptium.net/2022/10/a-short-exploration-of-java-class-pre-initialization/) with Alibaba and Google. 
- **[Migration toolkit for Java](https://projects.eclipse.org/projects/adoptium.emt4j)**. Most recent project, it shows how to manage some organizations' difficulties from Java 8 to Java 11+. The Alibaba team led the project as they wanted to share this tool. 

### Focus Areas for 2023, Adoptium Working group has four pillars for 2023:	
 - **Vibrant Healthy projects and Community**. Continue building a solid and healthy community. Attend events, and promote Adoptium.

>  - **Increase Product Usage**. keep up with trends, and ensure we build what people need. Explain the real value of our products. We already have the “we produce high-quality binaries”. We want to show how these binaries are produced, talk with the organizations, and ensure they don’t put their product at risk.

Reviewer 3: ```suggestion
 - **Increase Product Usage**. Keep up with trends, and ensure we build what people need. Explain the real value of our products. We already have the “we produce high-quality binaries”. We want to show how these binaries are produced, talk with organizations, and ensure they understand the project's position.
```

 - **Growing membership and involvement**. Participatory community, join the WG and bring your point of view. We need the voice of Java users and enterprises. It helps to bring additional funds. New opportunities, ensuring we have involvement from a wide variety of industries, have a robust continuity. 

### Working Group and Community update. 
Here we showed some ideas and opened a discussion on how we can promote community, working group, and sponsorship:
 - **New members**. We want to hear about Enterprise pain points and be sure how we add value to them. Reach out to industry decision-makers by doing B2B Marketing 
 - **Student programs**. We have collaborated with universities in the past. We are running the Outreachy program with two spaces: 1 founded by Eclipse and one by Red Hat.
 - **More Events hosted by Adoptium**.  Virtual conferences or being an active part of java community events. This is a line of direct sponsorship as well. 
 - **Adopter page**. We could have an adopter page on our website for all users. It doesn’t matter if they belong to Eclipse Adoptium WG. 

> ### Adoptium website.

Reviewer 3: ```suggestion
### Adoptium website
```

Hosted and maintained by the community and Eclipse Foundation team. It was developed in a very open way and translated into various languages. For 2023, we want to ensure that it helps us achieve our focus areas and promotes our initiatives very easily, according to the audience profile. 

### Adoptium Infrastructure

> The current infrastructure is composed of  300+ machines. The team is working on Infrastructure as code, as we rely on vendor collaborations and sponsorship. In this area, we need to be leaner and more efficient. The highest usage of machines is during release time, which happens every three months.  Our technical project management team is looking for collaborations in this area, as Adoptium does not have an Infra team. 

Reviewer 3: ```suggestion
The current infrastructure is composed of  300+ machines. The team is working on Infrastructure as code, as we rely on vendor collaborations and sponsorship. In this area, we need to be leaner and more efficient. The highest usage of machines is during release time, which happens every three months.  Our technical project management team is looking for collaborations in this area, as Adoptium is always looking to adopt best practices in the industry. 
```

### Reproducible builds

> To be compliant with Software Supply chain security. Currently, Temuring and AQAvit is compliance with SLSA level 2, and as part of the 2023 plan, we want to achieve level 4. This job is also done by having SBOM implemented (Software bill of materials) 

Reviewer 3: ```suggestion
An important part of the software supply chain security. Currently, Temurin is compliant with SLSA level 2, and as part of the 2023 plan, we want to achieve level 4. This includes a number of requirements including having SBOM implemented (Software bill of materials) and forms part of Adoptium's [secure software development](https://adoptium.net/en-GB/docs/secure-software/) initiative.
```

## Tuesday, the 25th: Eclipse Adoptium and OpenJDK Panel and Videos recording

> Again, our community representatives, Tim, George, and Shelley, together with Jan Westerkamp and me as a community manager had 35 minutes of the open panel to talk with EclipseCon members by giving a summary of all our main projects and explaining the 2023 plan. It was a good opportunity to hear from enterprise users about how and why they use Temurin over other distribution systems and get some ideas on how to keep people informed about project updates. We want to thank all the attendees! 

Reviewer 3: ```suggestion
Again, our community representatives, Tim, George, and Shelley, together with Jan Westerkamp and me as a community manager had 35 minutes of the open panel to talk with EclipseCon members by giving a summary of all our main projects and explaining the 2023 plan. It was a good opportunity to hear from enterprise users about how and why they use Temurin over other distributions, and get some ideas on how to keep people informed about project updates. We want to thank all the attendees! 
```

> An important part of the software supply chain security. Currently, Temurin is compliant with SLSA level 2, and as part of the 2023 plan, we want to achieve level 4. This includes a number of requirements including having SBOM implemented (Software bill of materials) and forms part of Adoptium's [secure software development](https://adoptium.net/en-GB/docs/secure-software/) initiative.

Reviewer 2: @tellison just checking on this statement. Is achieving SLSA Level 4 part of the 2023 program plan?

Author: Yes, it is part of the project priorities under "Secure Supply Chain and Software Engineering" topic.

![Eclipse Adoptium Panel](eclipse-aopdtium-panel.jpg)
*Tim, Shelley, Jan and George hosting Eclipse Adoptium panel*

We also used this day to record videos about Eclipse Adoptium 2023 plan and promote our main projects. Stay tuned to watch them. We thank the Jakarta EE working group for sharing this recording space. 

## Wednesday 25:  How I Learned to Stop Worrying and Love the SBOM - by Shelley Lambert and Jakarta EE reception sponsored by IBM

> In the morning, Shelley gave a talk explaining how Eclipse Adoptium is implementing SBOM (Software bill of materials) and why. Considering the audience, we realized it is a topic of interest as it is also linked to Eclipse foundation's goals for 2023 of achieving a Security supply chain for all their projects.

Reviewer 3: ```suggestion
In the morning, Shelley gave a talk explaining how Eclipse Adoptium is implementing SBOM (Software bill of materials) and why. Considering the audience, we realized it is a topic of interest as it is also linked to Eclipse foundation's goals for 2023 of achieving a security supply chain for all their projects.
```

> Again, our community representatives, Tim, George, and Shelley, together with Jan Westerkamp and me as a community manager had 35 minutes of the open panel to talk with EclipseCon members by giving a summary of all our main projects and explaining the 2023 plan. It was a good opportunity to hear from enterprise users about how and why they use Temurin over other distributions, and get some ideas on how to keep people informed about project updates. We want to thank all the attendees! 

Reviewer 2: `s/and me as a community manager/and myself as the community manager/`

> Again, our community representatives, Tim, George, and Shelley, together with Jan Westerkamp and me as a community manager had 35 minutes of the open panel to talk with EclipseCon members by giving a summary of all our main projects and explaining the 2023 plan. It was a good opportunity to hear from enterprise users about how and why they use Temurin over other distributions, and get some ideas on how to keep people informed about project updates. We want to thank all the attendees! 

Reviewer: ```suggestion
Again, our community representatives, Tim, George, and Shelley, together with Jan Westerkamp and myself as the community manager held a 35-minute open panel to talk with EclipseCon attendees by summarising all our main projects and explaining the 2023 plan. It was an excellent opportunity to hear from enterprise users about how and why they use Temurin over other distributions and get some ideas to keep people informed about project updates. We want to thank all the attendees! 
```

![SBOM talk by Shelley](sbom-talk-by-shelley.jpg)
*Shelley explaining what is an SBOM*

> The day closed with the Jakarta EE reception sponsored by IBM, and they invited me to give some words on the name of the community we represented. It was also a good moment to meet other sponsors and working group members.

Reviewer 3: ```suggestion
The day closed with the Jakarta EE reception sponsored by IBM, and they invited me to give some words on behalf of the community we represented. It was also a good moment to meet other sponsors and working group members.
```

![Jakarta EE IBM receptiom](jakartaee-ibm-reception.jpg)
*Carmen giving a thank you toast to both Jakarta EE and IBM team*

> ## Thursday 26: Clouser and Hackathon.

Reviewer 3: "Clouser"?

Author: typo: closure

Reviewer 3: Suggested a fix

> ## Thursday 26: Clouser and Hackathon.

Author: ```suggestion
## Thursday 26: Closure and Hackathon.
```

> We stayed around to have the opportunity and coordinate some strategic meetings with different Eclipse Foundation and Adoptium team members.

Reviewer: ```suggestion
We had an opportunity to coordinate some strategic meetings with different Eclipse Foundation and Adoptium team members.
```

 - **[Eclipse Temurin](https://adoptium.net/temurin/releases/)**. the flagship of the Adoptium projects, this brand name was created to separate our main product name from the Adoptium brand, and is an anagram of "RUNTIME".  It is a well-tested Java SE binary available across multiple platforms. Consumable via [the Adoptium website](https://adoptium.net/temurin/releases/), [API](https://api.adoptium.net/), [packages](https://packages.adoptium.net/), [container images](https://hub.docker.com/_/eclipse-temurin), [installers](https://adoptium.net/temurin/releases/), and third-party redistributors. Signed for safe OS use and with a $ free-to-use open source license. According to our download metrics, we have over [300k downloads daily](https://dash.adoptium.net), but we don't track who is downloading (due to GDPR). Our premise is that if users need and make the case we will keep building it, but every time we release, we need more time and infra support to maintain all the versions. Some ideas we got from the community:
    - Be more clear about promoting the advantages of using Temurin and certifications, compliance, and licenses.

>     - Define methods how to get the community and promote Temurin.

Reviewer 3: ```suggestion
    - Define methods to notify the community of new releases and promote Temurin.
```

> ## Thursday 26: Clouser and Hackathon.

Reviewer 3: ```suggestion
## Thursday 26: Closure and Hackathon.
```
